### ISSUE TITLE
Sip Analyzer Question re: Names versus IP addresses

### ISSUE THREAD

**Issue by bfortune2** [BroadSoft-Xtended/BroadWorks-Dashboards-and-Discovery#81]
Is there a way to update the ip address labels in the Kibana Sip Analyzer to actual names rather than IP addresses?

**Comment by pgawali-broadsoft** [comment]
Yes, you need to add host-name with IP in Elasticsearch directly.

Steps to add in Elasticsearch:
1. Mapping :
curl -X PUT "localhost:9200/servermapping?pretty" -H 'Content-Type: application/json' -d'
{
   "mappings":{
      "hostname":{
         "properties":{
            "server_ip":{"type":"keyword"},
            "hostname":{"type":"keyword"}
         }
      }
   }
}
'
2.Add Hostname and IP(You can add all your entries):

e.g.
curl -X PUT "localhost:9205/servermapping/hostname/2" -H 'Content-Type: application/json' -d'
{
    "server_ip" : "<Server_IP_Address>","hostname": "USER_A"
}
'
curl -X PUT "localhost:9205/servermapping/hostname/8" -H 'Content-Type: application/json' -d'
{
    "server_ip" : "Execution Server","hostname": "AS"
}
'

Thanks,
-Parikshit

[https://www.cisco.com/c/dam/m/en_us/signaturetool/images/banners/standard/07_standard_graphic.png]
Parikshit Gawali
ENGINEER.SOFTWARE ENGINEERING
pgawali@cisco.com<mailto:pgawali@cisco.com>
Tel:




Cisco Systems, Inc.



India
cisco.com
[http://www.cisco.com/assets/swa/img/thinkbeforeyouprint.gif]
Think before you print.
This email may contain confidential and privileged material for the sole use of the intended recipient. Any review, use, distribution or disclosure by others is strictly prohibited. If you are not the intended recipient (or authorized to receive for the recipient), please contact the sender by reply email and delete all copies of this message.
Please click here<http://www.cisco.com/c/en/us/about/legal/terms-sale-software-license-agreement/company-registration-information.html> for Company Registration Information.
